CONTRIBUTING.md

```diff
@@ -6,8 +6,10 @@ If you would like to contribute enhancements or fixes, please do the following:
 
 1.  Fork the plugin repository
 2.  Run `npm install` to setup all dependencies
-3.  Hack on a separate topic branch created from the latest `master`
-4.  Commit and push the topic branch
+3.  Hack on a separate topic branch created from the latest `master`. The src files are under `/src`
```

Comment:
This second sentence is repeating itself, maybe change it to something along the lines of:

```markdown
Changes to the package code should be made to the files in the `src` directory.
```